src/IOConnection.cpp

```diff
@@ -85,15 +86,22 @@ namespace eipScanner {
 		}
 	}
 
-	bool IOConnection::notifyTick(std::chrono::milliseconds period) {
-		auto periodInMicroS = period.count() * 1000;
+	bool IOConnection::notifyTick() {
```

Comment:
This change is fine for me, but I'd like t know your motivation?

Reply:
I have a class that contains an instance of ConnectionManager. It is created at program launch, so when I execute ConnectionManager::handleConnections() method, saved _lastHandleTime is outdated regarding current time, causing new forward open connections to be immediatly be closed. 

Maybe another way is to let _lastHandleTime un-initialized and test for its validity at first connection? And set it back un-initialized when forward close is called?

Reply:
No, let's use your solution. I wanted to spare some time not to calculate the period between two calls of `notifyTick` in each connection. However, I think your solution is more robust and clean.